nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/factory/Nd4j.java

```diff
@@ -5043,37 +5407,47 @@ public static INDArray createSparseCOO(DataBuffer values, DataBuffer indices, lo
         return matrix;
     }
 
-
     ////////////////////// OTHER ///////////////////////////////
 
-
-
     /**
-     * Creates a row vector with the specified number of columns
+     * Creates a 2D array with specified number of rows, columns initialized with zero.
      *
-     * @param rows    the rows of the sndarray
-     * @param columns the columns of the ndarray
-     * @return the created ndarray
+     * @param rows    number of rows.
+     * @param columns number of columns.
+     * @return the created array.
      */
     public static INDArray zeros(long rows, long columns) {
-        INDArray ret = INSTANCE.zeros(rows, columns);
-        return ret;
+        return  INSTANCE.zeros(rows, columns);
     }
 
     /**
-     * Creates a row vector with the specified number of columns
+     * Creates a 1D array with the specified number of columns initialized with zero.
      *
-     * @param columns the columns of the ndarray
-     * @return the created ndarray
+     * @param columns number of columns.
+     * @return the created array
      */
     public static INDArray zeros(int columns) {
         return INSTANCE.zeros(columns);
     }
 
+    /**
+     * Creates a 1D array with the specified data tyoe and number of columns initialized with zero.
```

Comment:
`specified data tyoe`
`number of columns initialized with zero.` -> rephrase, sounds like number of columns will be initialized to zero.
"number of columns, with values initialized to zero"